applications/nrf5340_audio/src/bluetooth/bt_stream/broadcast/broadcast_sink.c

```diff
@@ -44,6 +44,7 @@ struct audio_codec_info {
 struct active_audio_stream {
 	struct bt_bap_stream *stream;
 	struct audio_codec_info *codec;
+	uint32_t pd;
```

Comment:
Suggest we rename to pres_dly or pres_delay

Reply:
The struct member in base is called pd, that's why I chose the same name for this.